index.html

```diff
@@ -1799,6 +1824,14 @@ <h3 id="applying">
           next-most-appropriate icon as determined by examining the <a>image
           object's</a> members.
         </p>
+        <p class="note">
+          In order to use <a>adaptive icons</a>, you define the <code>foreground</code>
```

Comment:
Yeah, no `code`. ReSpec will automatically "code" it for you. If it doesn't, that's a bug somewhere else.